plugins/trezor/cmdline.py

```diff
@@ -1,7 +1,22 @@
 from electroncash.plugins import hook
-from .trezor import TrezorPlugin
+from electroncash.i18n import _
+from electroncash.util import print_stderr
+from .trezor import TrezorPlugin, PASSPHRASE_ON_DEVICE
 from ..hw_wallet import CmdLineHandler
 
+class TrezorCmdLineHandler(CmdLineHandler):
+    def __init__(self):
+        self.passphrase_on_device = False
```

Comment:
Do we want to enable this by default? Electrum has it set to false.

Reply:
I.. do not know. I personally hate passphrases... *shrug*